cmd/server/main.go

```
@@ -92,7 +95,10 @@
~	}
 }
 
 func (app *ServerApp) run(ctx context.Context) error {
+	ctx, cancel := context.WithCancel(context.Background())
```

golangci-lint (1.62.2)

[error] SA4009(related information): assignment to ctx

(staticcheck)